### ISSUE TITLE
Epic?

### ISSUE THREAD

**Issue by SamanthaWamtha** [KingKrouch/21xMachi9#2]
I got the game for free on epic and Idk if I just installed it wrong or something but it's not working for me

**Comment by velocity7** [comment]
I can confirm Epic Games Store version isn't working with this mod, Epic Games Store version is at 1.0.5.

**Comment by jaguilar** [comment]
Same here. Sadness! Game does not launch with the plugins installed and I can't seem to locate any log files.

**Comment by Naitrate** [comment]
Yeah, I can confirm that at the moment, this doesn't work with v1.05 of the game. My Attention Deficit issues are getting the worst of me, and because of how mentally draining it is to adjust memory value pointers each game update (I felt burned out by the time DBZK got done with game updates), I'm working on figuring out a better system that utilizes pattern scanning instead of hardcoded pointers.

I'm working on putting together a new solution of sorts which will put all of my Unreal Engine 4 game fixes under a single umbrella (That way, I can just make new plugins for each game rather than refactoring a dozen different visual studio solutions when I do general updates).

Apologies for the bad news, I'll try and get to it soon.

**Comment by jaguilar** [comment]
No problem over here. Just wanted to report!

**Comment by Isaiahmag** [comment]
Glad to see this is still getting worked on and hope to play in glorious ultrawide.

**Comment by Dldoshwaggins** [comment]
Just wanted to start playing this game, sitting on EPIC but also won't get it to work. Hope it will someday be ultrawide enjoyable! Best regards

**Comment by moranetavora** [comment]
Just commenting to be updated in case of correction.
